pkg/cmd/cli/backuplocation/create.go

```diff
@@ -148,6 +152,16 @@ func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
 		validationFrequency = &metav1.Duration{Duration: o.ValidationFrequency}
 	}
 
+	if len(o.Credential.Data()) > 1 {
```

Comment:
I think it would be better if we could add this check into the `Validate` function above. We don't seem to have an equivalent function for the `set` command so that would be good to add and use as the place to check in the other file.

Reply:
Since the `set` and `create` commands have separate `Options` types, we'll likely have to split the `Credentials` field out somehow to have a shared validation if that's what we want. But I otherwise agree on having a `Validate` function that does this!

Reply:
When I made this comment, introducing an equivalent `Validate` function was all I had in mind so I'm happy with that approach!

Reply:
Nolan and I paired and looking into merging some of the code from create and set, and it did not make sense.